package.json

```diff
@@ -45,11 +45,14 @@
     "after": "~0.8.1",
     "browzers": "1.2.0",
     "bulk-require": "0.2.1",
+    "dependency-check": "^2.10.0",
+    "electron": "^1.8.2",
+    "electron-prebuilt": "^1.4.13",
     "mocha": "~1.16.2",
+    "phantomjs": "^2.1.7",
     "phantomjs-prebuilt": "2.1.13",
     "tape": "3.5.0",
-    "through2": "0.6.3",
-    "zuul-ngrok": "^4.0.0"
```

Comment:
Must keep this one (for now). The reason that `dependency-check` doesn't see it, is because it's `require`d dynamically.